cmd/cspc-operator/app/storagepool_create.go

```diff
@@ -93,6 +94,7 @@ func (pc *PoolConfig) CreateStoragePool() error {
 }
 
 func (pc *PoolConfig) createCSP(csp *apis.CStorPoolInstance) (*apis.CStorPoolInstance, error) {
```

Comment:
changing `csp` to `cspi` whenever we touch the function can be better

Reply:
I will raise a separate PR specifically for this. I think I will have some cycle to do this.

Reply:
I will raise a separate PR specifically for this. I think I will have some cycle to do this